src/app/index.js

```diff
@@ -37,6 +37,12 @@ angular.module('app', [
         controller: 'EventController',
         controllerAs: 'vm'
       })
+      .state('register', {
+        url: '/register',
+        template: require('./components/register/registerView.html'),
+        controller: 'RegisterController',
+        controllerAs: 'vm'
+      })
```

Comment:
Same as the other commit. Let's start giving these names like `controllerAs: registerCtrl`.